src/snagrecover/templates/rockchip-spl-dfu.yaml

```diff
@@ -0,0 +1,8 @@
+code471:
+  path: mkimage-in-simple-bin.mkimage-u-boot-tpl
+  delay: 1
+code472:
+  path: mkimage-in-simple-bin.mkimage-u-boot-spl
```

Comment:
As noted earlier, these example paths is for temporary files that should probably not be considered output artifacts from a U-Boot build.